feat(infra): add SyncPutQueuePolicyTest test class #1336

rnmb opened the pull request:
1. add SyncPutQueuePolicyTest test class

Fixes #1336

Changes proposed in this pull request:
- 

> Check mailbox configuration when submitting. [Contributor Guide](https://hippo4j.cn/community/contributor-guide)

rnmb committed:
feat(infra): add SyncPutQueuePolicyTest test class #1336
1. add SyncPutQueuePolicyTest test class

Co-authored-by: rnmb<19276209+rnmb@users.noreply.github.com>

magestacks commented on infra/common/src/test/java/cn/hippo4j/common/executor/support/SyncPutQueuePolicyTest.java:
```diff
@@ -0,0 +1,34 @@
+package cn.hippo4j.common.executor.support;
```
Add the open source agreement to the top, and leave a blank line with package

magestacks commented on infra/common/src/test/java/cn/hippo4j/common/executor/support/SyncPutQueuePolicyTest.java:
```diff
@@ -0,0 +1,34 @@
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.concurrent.*;
```
Do not import *

magestacks commented on infra/common/src/test/java/cn/hippo4j/common/executor/support/SyncPutQueuePolicyTest.java:
```diff
@@ -0,0 +1,34 @@
+
+        Assert.assertSame(syncPutQueuePolicy, threadPoolExecutor.getRejectedExecutionHandler());
+        IntStream.range(0, 4).forEach(s -> {
+            threadPoolExecutor.execute(() -> ThreadUtil.sleep(500L));
```
Shorten the sleep time, otherwise it will affect the project build time

rnmb committed:
feat(infra): SyncPutQueuePolicyTest #1336
1. Add the open source agreement to the top, and leave a blank line with package

Co-authored-by: rnmb<19276209+rnmb@users.noreply.github.com>
